google-cloud-storage/CHANGELOG.md

```diff
@@ -1,5 +1,21 @@
 # Release History
 
+### 1.21.0 / 2019-08-15
+
+#### Features
+
+* Support overriding of service endpoint
+
+* Update documentation
+
```

Comment:
Seems like too many blank lines in this section, although "Bug Fixes" below looks good.